portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+/* toggles the drop down text */
```

Comment:
General recommendations for comments:
- Use consistent capitalization everywhere. So start all the comments with a caps letter.
- End all with a "." or don't end any that way. For example: The top comment ends with "License.", but all these new ones don't.

Reply:
Capitalized everything. End each line with .